src/ApiPort/ApiPort.VisualStudio/ApiPort.VisualStudio.vsct

```diff
@@ -83,7 +83,7 @@
         <CommandFlag>DynamicVisibility</CommandFlag>
         <CommandFlag>DefaultInvisible</CommandFlag>
         <Strings>
-          <ButtonText>Analyze Assembly Portability</ButtonText>
+          <ButtonText>Analyze Solution Portability</ButtonText>
```

Comment:
Looks like this menu button goes to two places, one is under "Analyze" menu, the other is when right click on the solution. This change is applicable well for later, but may not for former because it can point to any assembly, which doesn't have to be the current solution.